src/sdfv.ts

```diff
@@ -82,7 +167,7 @@ function init_sdfv(
         file = e.target.files[0];
         reload_file();
     });
-    $('#menuclose').on('click', () => close_menu());
+    $('#menuclose').on('click', () => SDFV.get_instance().close_menu());
```

Comment:
not a big fan of this as there may be multiple SDFV instances open in the same browser.

Reply:
You mean the same HTML document, i.e. the same browser window and tab? Because if you have multiple tabs of SDFV open, that's not an issue.

Reply:
I mean in a jupyter notebook for example